packages/kolme-store/src/error.rs

```diff
@@ -20,14 +24,39 @@ pub enum KolmeStoreError {
     // kolme#144 - Reports a double insert (Block already exists with same hash and insert)
     #[error("Block already in database: {height}")]
     MatchingBlockAlreadyInserted { height: u64 },
+
     #[error("Transaction is already present in database: {txhash}")]
     TxAlreadyInDb { txhash: Sha256Hash },
+
+    #[error("get_height_for_tx: invalid height in Fjall store: {details}")]
+    InvalidHeightInFjall { details: String },
```

Comment:
It would be better to provide structured error info within these values, not just `String`s. Is there a reason to stick with strings here?